### ISSUE TITLE
Q: What does it mean to "serve drinks"?

### ISSUE THREAD

**Issue by benatuts** [RoboCupAtHome/RuleBook#570]
In Serving Drinks...

I'm not entirely clear of the role of the bartender. Am I correct in understanding that all available drinks are placed on the bar, and that the bartender, if asked, will place that drink in the robot's gripper or tray?

Also, may two or more people share the same drink preference? If so, are there unlimited supplies of each available drink, one drink for each beverage, or does the "available beverages" on the bar mean the entire stock of the bar and that there may be a finite multiple of each. In which case, are there penalties for asking about available quantities?

**Comment by johaq** [comment]
Yes, all drinks will be on the bar. Bartender can be instructed by the robot to handover drinks or place them in a tray or the robot can get them itself.
Yes, one drink is not available, supply for others is infinite.

**Comment by RemiFabre** [comment]
Will the drinks be part of the "regular known objects" discovered during the setup days?

**Comment by kyordhel** [comment]
# TL;DR; Yes

## Rule of thumb:
- The robot must recognize that object or its properties to interact with it: Known/Alike
- The robot may encounter the object and avoid it or dispose it: Known/Alike/Unknown
- The robot must describe an object based on its properties: Unknown

**Comment by kyordhel** [comment]
@benatuts if your question has been answered/solved, please close the issue.

**Comment by kyordhel** [comment]
Expired.
